client/src/components/Messages/ChatSideBanner/ChatSideBanner.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+import SideBannerItem from '../sideBannerItem/SideBannerItem';
+import { useConversation } from '../../../context/useConversationContext';
+import { useSocket } from '../../../context/useSocketContext';
+import { useEffect, useState } from 'react';
+
+const ChatSideBanner = (): JSX.Element => {
```

Comment:
A lot of this code seems to be from a previous PR, please adjust the base of the PR or rebase to only reflect the changes :)

Reply:
@navjot-chahal please see comment above